src/java.desktop/share/classes/com/sun/java/swing/plaf/gtk/GTKFileChooserUI.java

```diff
@@ -432,6 +432,18 @@ public  DoubleClickListener(JList<?> list) {
         public void mouseClicked(MouseEvent e) {
```

Comment:
The same issue has been observed in case of key navigation also. we need to handle those scenario where a user is navigating through keys. I will check and update accordingly as we discussed over call.